ai_agents/devable_research_agent/skills/tracking-list/scripts/collect.py

```
@@ -0,0 +1,307 @@
+            executor.submit(func, config, from_date, to_date, depth): name
+            for name, func in active_collectors.items()
+        }
+
+        for future in as_completed(futures):
+            name = futures[future]
+            try:
+                result = future.result(timeout=600)
+                collection_results.append(result)
+                all_items.extend(result.items)
+                _log(f"[{name}] Done: {len(result.items)} items, {len(result.errors)} errors")
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)